etc/p3-config/p3-config.yml

```diff
@@ -603,8 +603,7 @@ p3_image_mlnx_ofed:
     - "vm"
     - "systemd-modules-load"
   env:
-    DIB_MLNX_OFED_VERSION: "4.1-1"
-    DIB_MLNX_OFED_REPO: "{{ p3_repo_server_url_mlnx_ofed }}"
+    DIB_MLNX_OFED_REPO: "https://linux.mellanox.com/public/repo/mlnx_ofed/latest/rhel7.6/mellanox_mlnx_ofed.repo"
```

Comment:
Don't you want https://linux.mellanox.com/public/repo/mlnx_ofed/latest/rhel7.6/x86_64/?

Reply:
No